linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,62 @@
+name: nextcloud-client
+version: 2.2.4-git
+summary: nextcloud desktop client
+icon: ../client/theme/colored/owncloud-icon.svg
+description: |
+  The ownCloud Desktop Client is a tool to synchronize files from ownCloud
+  Server with your computer.
+
+grade: devel
+confinement: strict
+
+parts:
+  client:
+    plugin: cmake
+    source: ../
+    source-subdir: client
+    build-packages:
+      - libqt5webkit5-dev
+      - libsqlite3-dev
+      - libssl-dev
+      - pkg-config
+      - qt5keychain-dev
+      - qttools5-dev-tools
+    configflags:
+      - -DCMAKE_INSTALL_PREFIX=/usr
+      - -DCMAKE_INSTALL_LIBDIR=/usr/lib
+      - -DDOEM_THEME_DIR=../src/nextcloudtheme
+    after:
+      - desktop-qt5
+      - indicator-qt5
```

Comment:
Take a look at [this](https://github.com/kyrofa/nextcloud-client-snap) (and test with `sudo snap install --edge nextcloud-client-kyrofa on amd64) to see what I did specifically. I'm curious what this part adds on top of that. As I mentioned, I couldn't get this to run, so I was unable to see for myself.